docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -157,7 +157,7 @@ The `config` object accepts the following parameters:
 | `locales`       | Array of Strings | Defines availables locales (see [updating locales](/developer-docs/latest/development/admin-customization.md#locales)) |
 | `translations`  | Object           | [Extends the translations](#extending-translations)                                                                                                                       |
 | `menu`          | Object           | Accepts the `logo` key to change the [logo](#logos) in the main navigation                                                           |
-| `theme`         | Object           | Overrides or [extends the theme](#theme-extension)                                                                                          |
+| `theme.light` and `theme.dark` | Object           | Override or [extend the theme](#theme-extension) for Light and Dark modes                                                         |
```

Comment:
I'd settle for "overwrite" here. While it is theoretically possible to extend the theme, one of our DS components can't make use of an extension. Therefore overwriting properties is probably the most common case.

Wdyt?

```suggestion
| `theme.light` and `theme.dark` | Object           | [Overwrite theme properties](#theme-extension) for Light and Dark modes                                                         |
```